### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mo-traor3-ai**
add bug report guidelines and "how to file a bug report" video

# Description

List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

-   [x] This change requires a documentation update

** No changes to the "code" - only additions made to the `CONTRIBUTING.md` file

## Any specific deployment considerations

For example, documentation changes, usability, usage/costs, secrets, etc.

## Docs

-   [x] Docs updated? What were the changes: updated rules/info on how to submit a bug in the `CONTRIBUTING.md` file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
...
+a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better
+understand and diagnose your problem.
+
+### How to File a Bug Report (VIDEO)
+[![Roboflow Support: Filing Bug Reports](https://img.youtube.com/vi/KE_SsEPaR_4/0.jpg)](https://www.youtube.com/watch?v=KE_SsEPaR_4)
+
+## License
+
+By contributing, you agree that your contributions will be licensed under
+the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)
```
Comment (SkalskiP) [inline CONTRIBUTING.md:46]: I'm not sure about the license. @yeldarby what do you think? Do we add any licenses to the notebooks repository? It feels weird, given the fact we mainly show how to run someone else's code.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
...
+- ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this
+  repository. [Roboflow](https://roboflow.com/) does not provide support for custom code ⚠️. If custom code has been added, please post in the Community Forum for more assistance: https://discuss.roboflow.com
+
+If you believe your problem meets all of the above criteria, please close this issue and raise a new one using the 🐛
+**Bug Report** [template](https://github.com/roboflow/notebooks/issues/new/choose) and providing
+a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better
+understand and diagnose your problem.
+
+### How to File a Bug Report (VIDEO)
+[![Roboflow Support: Filing Bug Reports](https://img.youtube.com/vi/KE_SsEPaR_4/0.jpg)](https://www.youtube.com/watch?v=KE_SsEPaR_4)
```
Comment (SkalskiP) [inline CONTRIBUTING.md:41]: Video is a great addition, but we need to make the video thumbnail more eye-catching.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
...
+- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem
+
+In addition to the above requirements, for the [Roboflow](https://roboflow.com/) team to provide assistance your code
+should be:
+
+- ✅ **Current** – Verify that your code is up-to-date with current
+  GitHub [main](https://github.com/roboflow/notebooks/tree/main), and if necessary `git pull` or `git clone` a new
+  copy to ensure your problem has not already been resolved by previous commits.
+- ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this
+  repository. [Roboflow](https://roboflow.com/) does not provide support for custom code ⚠️. If custom code has been added, please post in the Community Forum for more assistance: https://discuss.roboflow.com
```
Comment (SkalskiP) [inline CONTRIBUTING.md:33]: We could also add a link to [GitHub Discussions](https://github.com/roboflow/notebooks/discussions). I, for one, will be following the post here more than the Forum.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
 - When adding or editing notebooks, make sure to specify the versions of pip packages 📦 you are using.
 
 Please keep in mind that repository is still under construction 🚧. We will add more contribution guidelines soon.
+
+## Submitting a Bug Report 🐛
```
Comment (SkalskiP) [inline CONTRIBUTING.md:10]: Could we also provide people with some guidelines on Feature Requests?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
...
+  copy to ensure your problem has not already been resolved by previous commits.
+- ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this
+  repository. [Roboflow](https://roboflow.com/) does not provide support for custom code ⚠️. If custom code has been added, please post in the Community Forum for more assistance: https://discuss.roboflow.com
+
+If you believe your problem meets all of the above criteria, please close this issue and raise a new one using the 🐛
+**Bug Report** [template](https://github.com/roboflow/notebooks/issues/new/choose) and providing
+a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better
+understand and diagnose your problem.
+
+### How to File a Bug Report (VIDEO)
```
Comment (SkalskiP) [inline CONTRIBUTING.md:40]: I'd drop the header.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
 - When adding or editing notebooks, make sure to specify the versions of pip packages 📦 you are using.
 
 Please keep in mind that repository is still under construction 🚧. We will add more contribution guidelines soon.
```
Comment (SkalskiP) [inline CONTRIBUTING.md:8]: Let's drop that line

**Review by SkalskiP — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (mo-traor3-ai) [reply CONTRIBUTING.md:46]: I set that license to match the ones on our other main repos, such as the python package. If it needs to be changed, I can do that

File: CONTRIBUTING.md (hunk 5 above)
Comment (mo-traor3-ai) [reply CONTRIBUTING.md:40]: just removed it, it'll be updated in the new PR

File: CONTRIBUTING.md (hunk 6 above)
Comment (mo-traor3-ai) [reply CONTRIBUTING.md:8]: just removed it, it'll be updated in the new PR

File: CONTRIBUTING.md (hunk 3 above)
Comment (mo-traor3-ai) [reply CONTRIBUTING.md:33]: forsure - including that as well in the new PR

**Comment by mo-traor3-ai** [comment]
Added requested changes - I've left the License conversation unresolved in case there's more to learn there from Product team

**Pull request closed without merging**